Solutions/Corvus.Json.PropertyBag.Benchmarks/Corvus.Json.PropertyBag.Benchmarks.csproj

```diff
@@ -0,0 +1,38 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <EndjinDisableAutoPackagesLock>true</EndjinDisableAutoPackagesLock>
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+  
+  <Import Project="$(EndjinProjectPropsPath)" Condition="$(EndjinProjectPropsPath) != ''" />
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+
+    <!--
+    CS1591, SA1600 - XML doc required. Not necessary for benchmark project
+    CA1816 - dispose pattern-related nonsense asking us to call GC.SuppressFinalizer in Dispose
+    -->
+    <NoWarn>$(NoWarn);CS1591;SA1600;CA1816</NoWarn>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="BenchmarkDotNet" Version="0.14.0" />
+    <PackageReference Include="Corvus.Json.ExtendedTypes" Version="1.0.3" />
```

Comment:
The latest version of Corvus.Json.ExtendedTypes is 4.1.4, but when we upgrade to this it causes breaking changes.
Corvus.Json.ExtendedTypes 1.0.3 has known vulnerabilities via System.Text.Json 7.x

Reply:
It's only the benchmark project so it's not critical, but I've upgraded this to the latest JSON Schema bits.